java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java

```diff
@@ -1866,6 +1868,7 @@ public int deleteSystem(User loggedInUser, Integer serverId, String cleanupType)
             network.put("ip", StringUtils.defaultString(server.getIpAddress()));
             network.put("ip6", StringUtils.defaultString(server.getIp6Address()));
             network.put("hostname", StringUtils.defaultString(server.getHostname()));
+            network.put("fqdn", StringUtils.defaultString(server.getFqdn()));
```

Comment:
Since we are adding a new attribute here, should we make clear this is the primary one?

Reply:
I've renamed it from `fqdn` to `primary_fqdn`.